src/app/component/ExpandableContentShowMore/ExpandableContentShowMore.component.js

```diff
@@ -29,18 +29,47 @@ export class ExpandableContentShowMore extends PureComponent {
     };
 
     state = {
-        isOpen: false
+        isOpen: true
     };
 
+    isExpanding = false;
+
+    expandableContentHeight = 'auto';
+
+    expandableRef = createRef();
+
     ref = createRef();
 
-    handleShowAllButtonClick = () => {
-        const { pageYOffset } = window;
+    componentDidMount() {
+        this.expandableContentHeight = this.expandableRef.current.getBoundingClientRect().height;
+        this.setState({ isOpen: false });
+    }
 
-        this.setState(
-            ({ isOpen }) => ({ isOpen: !isOpen }),
-            () => window.scrollTo(0, pageYOffset)
-        );
+    getSnapshotBeforeUpdate() {
+        if (this.isExpanding) {
+            const { pageYOffset } = window;
+            const preventScrolling = () => window.scrollTo(0, pageYOffset);
+
+            const ONE_SECOND_IN_MS = 1000;
+            const transitionDurationCSStoMS = window
+                .getComputedStyle(this.expandableRef.current)
+                .getPropertyValue('transition-duration')
+                .slice(0, -1) * ONE_SECOND_IN_MS;
+
+            window.addEventListener('scroll', preventScrolling);
+            setTimeout(() => window.removeEventListener('scroll', preventScrolling),
+                transitionDurationCSStoMS);
+            this.isExpanding = false;
```

Comment:
Why is this done? So user can be scrolled back to the open attributes?

Reply:
Otherwise the page scrolls together with filter expanding/collapsing

Reply:
Found a way around it. It was a Chrome anchor scroll behavior problem. (See last commit)